src/main/webapp/app/overview/course-card.component.html

```diff
@@ -2,101 +2,90 @@
-                        <span class="next-exercise-title">{{ nextRelevantExercise.title }}</span>
-                    </div>
-                    <div class="col-3 next-exercise-col text-nowrap">
-                        @if (nextExerciseDueDate) {
-                            <div>
-                                <h6>{{ 'artemisApp.exercise.dueDate' | artemisTranslate }}:</h6>
-                                <h6>{{ nextExerciseDueDate | artemisTimeAgo }}</h6>
+                    <div class="col-7" [ngClass]="{ 'col-12': !(exerciseCount && (totalReachableScore > 0 || totalAbsoluteScore > 0)) }">
+                        @if (exerciseCount > 0 && (totalReachableScore > 0 || totalAbsoluteScore > 0)) {
+                            <div class="align-self-center">
+                                <div class="fw-bold text-secondary" jhiTranslate="artemisApp.studentDashboard.cardScore"></div>
+                                <div class="fw-bold">
+                                    <div
+                                        jhiTranslate="artemisApp.studentDashboard.points"
+                                        [translateValues]="{
+                                            totalAbsoluteScore: totalAbsoluteScore,
+                                            totalReachableScore: totalReachableScore,
+                                            totalRelativeScore: totalRelativeScore,
+                                        }"
+                                    ></div>
+                                </div>
+                            </div>
+                        } @else {
+                            <div class="fw-bold text-secondary" jhiTranslate="artemisApp.studentDashboard.cardScore"></div>
+                            <div class="fw-bold">
+                                <span jhiTranslate="artemisApp.studentDashboard.noStatistics"></span>
+                            </div>
+                        }
+                        <hr />
+                        <div class="fw-bold text-secondary" jhiTranslate="artemisApp.studentDashboard.cardExerciseLabel"></div>
+                        @if (nextRelevantExercise) {
+                            <a class="fw-bold btn-exercise exercise-title exercise-guided-tour" [routerLink]="['/courses', course.id!, 'exercises', nextRelevantExercise.id!]">
+                                {{ nextRelevantExercise.title }}
+                            </a>
+                        } @else {
+                            <div class="fw-bold mb-3 exercise-guided-tour">
+                                <span jhiTranslate="artemisApp.studentDashboard.cardNoExerciseLabel"></span>
                             </div>
                         }
                     </div>
+                    @if (exerciseCount > 0 && (totalReachableScore > 0 || totalAbsoluteScore > 0)) {
```

Comment:
You might be able to simplify this by leaving out `> 0`, e.g. for exerciseCount. Can you check?